.goreleaser.prerelease.yml

```diff
@@ -13,32 +9,21 @@ builds:
       - amd64
       - arm64
     ldflags:
-      - '-w -s -X github.com/chanzuckerberg/aws-oidc/pkg/util.GitSha={{.Commit}} -X github.com/chanzuckerberg/aws-oidc/pkg/util.Version={{.Version}} -X github.com/chanzuckerberg/aws-oidc/pkg/util.Dirty=false -X github.com/chanzuckerberg/aws-oidc/pkg/util.Release=true'
-
-dockers:
```

Comment:
aws-oidc has a docker server-side component so we'll need to bring this back in a subsequent change

Reply:
thanks for the reminder! completely forgot about that. I just added it back